### PR TITLE
global: `watchdog` package addition

### LINKED ISSUE #2778: global: flask app not reloading
As described here: http://stackoverflow.com/a/28242285/2230231, due to some changes in `Werkzeug 0.10`, now automatic changes detection is not working (unless it was my case in my `zenodo` local env and also @hachreak told me that he has the same issue) - it also affects noticeable increase of CPU usage.

The **proposed solution** (taking [this](https://github.com/mitsuhiko/werkzeug/blob/4e2c6746e565eaba7b3bd7489337580b62db610b/werkzeug/_reloader.py#L246-L251) into account) is to install [watchdog](https://pypi.python.org/pypi/watchdog).

Shall we add this somewhere in requirements?

### PR DESCRIPTION
**Pull request by bouzlibop**
* Enables automatic code changes detection. (closes #2778)

* Reduces CPU usage.

Signed-off-by: Adrian Pawel Baran <adrian.pawel.baran@cern.ch>
Reviewed-by: Jiri Kuncar <jiri.kuncar@cern.ch>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -132,6 +132,7 @@ def run(self):
     ],
     "development": [
         "Flask-DebugToolbar==0.9.0",
+        "watchdog==0.8.3"
```
Comment (jirikuncar) [inline setup.py]: Please add comma (`,`) at the end of the line so next time somebody adds a package we get again nice one line diff.

**Comment by jirikuncar** [comment]
@bouzlibop please fix a typo in commit message `useage` -> `usage`

**Comment by bouzlibop** [comment]
Thanks @jirikuncar. I just fixed these things you mentioned.

**Comment by kaplun** [comment]
Out of curiosity: how the watchdog package is used in Invenio? It is not referred anywhere.

**Comment by bouzlibop** [comment]
@kaplun, `flask` is based on `Werkzeug`, which is dealing (besides huge amount of other stuff) with automatic reloading on code changes and it's using `watchdog` if [available](https://github.com/mitsuhiko/werkzeug/blob/4e2c6746e565eaba7b3bd7489337580b62db610b/werkzeug/_reloader.py#L246-L251).

**Pull request merged**